duckduckgo/macos-browser
# More Xcode cleaning

Task/Issue URL:  https://app.asana.com/0/1201037661562251/1199661670583310/f
https://app.asana.com/0/1199230911884351/1201928755345685/f

**Description**:
This PR contains:

- Renaming of tooltip to tab preview in order to avoid confusion

- Unification of file group names in Unit Tests folder  

**Steps to test this PR**:
1. Make sure tab previews work
2. Make sure you can compile the product using all available compilation schemes

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
